sdks/python/apache_beam/io/gcp/pubsub.py

```diff
@@ -134,8 +136,13 @@ class ReadFromPubSub(PTransform):
   """A ``PTransform`` for reading from Cloud Pub/Sub."""
   # Implementation note: This ``PTransform`` is overridden by Directrunner.
 
-  def __init__(self, topic=None, subscription=None, id_label=None,
-               with_attributes=False, timestamp_attribute=None):
+  def __init__(self,
+               topic=None,  # type: typing.Optional[str]
```

Comment:
If we keep these, I wonder if it'd be easier to read if the comments were all aligned.

Reply:
I'll leave that up to the Beam Team, but we're still going to have the line length limit to contend with.

Reply:
fixed